django_spanner/introspection.py

```diff
@@ -254,15 +254,14 @@ def get_constraints(self, cursor, table_name):
 
         # Add the various constraints by type.
         constraint_types = cursor.run_sql_in_snapshot(
-            '''
+            """
             SELECT
                 CONSTRAINT_NAME, CONSTRAINT_TYPE
             FROM
                 INFORMATION_SCHEMA.TABLE_CONSTRAINTS
             WHERE
-                TABLE_NAME="{table}"'''.format(
-                table=quoted_table_name
-            )
+                TABLE_NAME=@table AND TABLE_SCHEMA=@schema_name""",
+            params={"table": quoted_table_name, "schema_name": schema_name},
```

Comment:
Here also: The table name should not be quoted when used as a parameter. Can we have a test for this method as well?

Reply:
Fixed as discussed